Adds Train Search protocol event range to the EventIdentifiers standard

balazsracz opened the pull request:
Adds a new section "Well-Known Other".
Places the train search protocol event range there.
Adds reference to the train search protocol standard.

bakerstu commented:
We need to add an update to the TN as well. At a minimum, the section heading needs to be added, it may have an empty body, however, putting some explanation text in the TN I think is desirable.

bakerstu commented:
> We need to add an update to the TN as well. At a minimum, the section heading needs to be added, it may have an empty body, however, putting some explanation text in the TN I think is desirable.

Some suggestions for the TN text:

> The general properties of the Well-Known Other Event Identifiers are the same as those for Well-Known Event Identifiers as described in section 2.5.3 above. In general, this section contains events that have not originated as Well-Known, typically in a private Unique Identifier space, but at a later time have been promoted to Well-Known status.

BTW, section 2.5.3 line 160 has a typo. It should be "manufacturer" with an 'r' suffix added. We could fix this while we are at it.

balazsracz commented:
> > We need to add an update to the TN as well. At a minimum, the section heading needs to be added, it may have an empty body, however, putting some explanation text in the TN I think is desirable.
> 
> Some suggestions for the TN text:
> 
> > The general properties of the Well-Known Other Event Identifiers are the same as those for Well-Known Event Identifiers as described in section 2.5.3 above. In general, this section contains events that have not originated as Well-Known, typically in a private Unique Identifier space, but at a later time have been promoted to Well-Known status.

added

> 
> BTW, section 2.5.3 line 160 has a typo. It should be "manufacturer" with an 'r' suffix added. We could fix this while we are at it.

fixed
